kivy/uix/textinput.py

```diff
@@ -3059,7 +3061,7 @@ def _set_text(self, text):
         self._refresh_text(text)
         self.cursor = self.get_cursor_from_index(len(text))
 
-    text = AliasProperty(_get_text, _set_text, bind=('_lines', ))
+    text = AliasProperty(_get_text, _set_text, bind=('_lines',), cache=True)
```

Comment:
`_lines_flags` is not a `Property`. Maybe that should be changed?

Reply:
Maybe, but the impact must be checked. I think it could be done in another PR just to test the impact of caching and changing _line_flags to property. Remove this one please :)